### DISCUSSION TITLE
Window Rule: Set Window Height

### DISCUSSION THREAD

**Discussion by garrettwp** [YaLTeR/niri#669] [Ideas]
It would be nice to be able to set the proportional height of a window via a window rule. Similar to default-column-width, have something like default-window-height. This would be useful for apps that look out of place at full height of the monitor. For example a calculator app, setting terminal to half height on a 4k monitor, etc.

**Comment by YaLTeR** [comment]
I think this should be better fixed by defaulting to float when that is implemented.
